### PR TITLE
Release notes for 0.67.x

### LINKED ISSUE #5449: Release notes for 0.67.x (Release Channel)
Release Notes 

 - Add detailed view of pending contributions.([#3175](https://github.com/brave/brave-browser/issues/3175))
 - Referrer spoofing could disable login CSRF protections on some sites.([#3422](https://github.com/brave/brave-browser/issues/3422))
 - Fix clock and stats padding for right-to-left languages on the New Tab Page.([#3574](https://github.com/brave/brave-browser/issues/3574))
 - Excluded sites can't be viewed without having them restored.([#3657](https://github.com/brave/brave-browser/issues/3657))
 - Retries for unverified Publishers .([#3741](https://github.com/brave/brave-browser/issues/3741))
 - Insufficient funds notification when there is no site in a-c list.([#4104](https://github.com/brave/brave-browser/issues/4104))
 - Update verified icons globally inside Rewards.([#4162](https://github.com/brave/brave-browser/issues/4162))
 - Restoring a wallet does not check for grant.([#4220](https://github.com/brave/brave-browser/issues/4220))
 - Add support for Twitter tips.([#4346](https://github.com/brave/brave-browser/issues/4346))
 - No visual confirmation that a verified publisher check is done in rewards panel.([#4351](https://github.com/brave/brave-browser/issues/4351))
 - Normalize callback should exclude 0% publishers.([#4386](https://github.com/brave/brave-browser/issues/4386))
 - Add Tweet button to every Tip panel.([#4393](https://github.com/brave/brave-browser/issues/4393))
 - Update default tipping banner text.([#4616](https://github.com/brave/brave-browser/issues/4616))
 - Ads Rewards should be refreshed after redeeming tokens.([#5011](https://github.com/brave/brave-browser/issues/5011))
 - Enable Brave ads for Australia, New Zealand and Ireland.([#5153](https://github.com/brave/brave-browser/issues/5153))
 - Failed payment balances do not retry unless browser is restarted or ad grant is claimed.([#5175](https://github.com/brave/brave-browser/issues/5175))
 - Upgrade from Chromium 76.0.3809.62 to Chromium 76.0.3809.72.([#5299](https://github.com/brave/brave-browser/issues/5299))
 - Errors related to TFuel collection on sliver.tv.([#5380](https://github.com/brave/brave-browser/issues/5380))

### PR DESCRIPTION
**Pull request by LaurenWags**
Fix #5449

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [x] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Test Plan:

n/a

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [0.67.123](https://github.com/brave/brave-browser/releases/tag/v0.67.123)
+
+ - Added the ability to opt-in to user private Brave ads for Australia, New Zealand and Ireland. ([#5153](https://github.com/brave/brave-browser/issues/5153))
+ - Added the ability to tip Twitter publishers. ([#4346](https://github.com/brave/brave-browser/issues/4346))
```
Comment (NejcZdovc) [inline CHANGELOG.md]: I would move tips to first place

Comment (LaurenWags) [reply CHANGELOG.md]: I debated between that and Ads. wdyt @kjozwiak ?

Comment (NejcZdovc) [reply CHANGELOG.md]: why I said this is because we will have PR around twitter, but not around ads

Comment (LaurenWags) [reply CHANGELOG.md]: yes, makes sense. will change the order.

**Pull request merged**